docker/misk-web/README.md

```diff
@@ -0,0 +1,80 @@
+squareup/misk-web
+===
+
+Base Alpine NodeJS package for Misk Web builds. Includes `misk-web` builder script and yarn cache hydrated with @misk/ NPM packages.
+
+`misk-web` handles running various commands over all @misk/ packages and tabs
+
+Example Usage
+---
+
+- Build on all tabs:
+  ```bash
+  $ docker run -d --rm --name container-name -v /absolute/local/path/to/web:/web squareup/misk-web:0.0.1 misk-web -b
```

Comment:
Not sure if this is integrated higher up the chain, but if devs are going to run these commands themselves, it might be nice to wrap them and make it as easy as possible to run. ie  ./build_all_tabs <path> <absoultely necessary params>  etc.

Reply:
For now that is built in to Gradle 👍